Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -274,7 +274,7 @@ static void WriteToHardware(u32 em_address, const u32 data, const u32 size)
 
   bool wi = false;
 
-  if (!never_translate && MSR.DR)
+  if (!never_translate && (IsOpcodeFlag(flag) ? MSR.IR : MSR.DR))
```

Comment:
What is an "opcode write"? My undestanding is that MSR[IR] only matters for instruction fetches.

Reply:
Oh, you know that's a fair point, didn't even think about that.